packages/components/src/components/as-range-slider/track/as-range-slider-bar.scss

```diff
@@ -17,7 +17,7 @@ as-range-slider-bar {
     background-color: var(--as--range-slider__range-bar--background-color--disabled);
   }
 
-  .as-range-slider__range-bar--draggable {
+  .as-range-slider__range-bar--isDraggable {
```

Comment:
You know, I'm totally OK with this but I wonder if someone would consider changing class names a breaking change :thinking: